requirements.adoc

```diff
@@ -53,17 +67,46 @@ specific patient.
 CONTEXT::
 The radiology order is sent to the radiology department for execution.
 
+
+Judy::
+
+```
+There is a use case where an outside image(done from a different facility is obtained and brought by the patient
+and needs to be read at the local facility).Therefore there is an order for "Outside Interpretation" which would include
+1. Patient Name (First , Middle and Last Name)/ MRN
+2. Study type eg CT head
+3. Name of facility
+4. Whether report is available..
+
+This would create a new study that is complete (after images are uploaded to the PACS system) and a user can claim a report. The label for this study would be
+OUTSIDE CT HEAD .. etc
+```
+
 COMPONENTS::
 Its main components are
 
-* patient
-* urgency
-* start date
+* patient details (MRN/Name)
+* urgency (These are priorities. Some people use numbers eg 0 for must be read immediately, 2 for ED studies , 4 for inpatients and 5 for outpatient /non emergent studies). These could be set up as a global property that can be changed by the person creating
+the order , and this field created as a drop down.
+* start date -- I would change this to Study Date...
```

Comment:
I dont think we should mix Order and Study, these are separate entities. An order is scheduled for day X but might be performed (the actual Study is made) on day Y which would be the study date.